docs/containers/create-first-webapp.md

```diff
@@ -42,11 +42,15 @@ Start a container with the `hello-world` website:
 ```shell
 docker run -d -p 8081:8080 webapp
 ```
+The `hello-world` page is now available at [http://127.0.0.1:8081](http://127.0.0.1:8081)
 
 Test the `hello-world` website using the `curl` command:
 
 ```shell
 curl http://127.0.0.1:8081
 ```
+Aftrer this you can kill container using:
+```shell
+docker stop (<container_ID> or <container_name>)
```

Comment:
Please add example how to get container ID using `docker ps` command $(docker ps -q -f name=<CONTAINER_NAME>)